force-app/main/default/classes/CON_ContactMerge_CTRL.cls

```diff
@@ -336,6 +333,7 @@ public with sharing class CON_ContactMerge_CTRL {
     /***********************************************************************************************
     * @description The rows to merge.
     */
+    @TestVisible
```

Comment:
Should these be switched to private if they're marked TestVisible?

Reply:
Agreed.  If they stay public then they don't need the test visible. If they're accessed only in test then they should be made private and test visible.

Reply:
The reason this has the annotation still even though it's public is because the `set` property is set to `private` so it functionally operates like a private property still and cannot be changed publicly without that annotation.

Reply:
I didn't want to make the entire property `private` since that would likely interfere with performing a `get` from the VF page.